lib/features/reps_core/reps_core.install

```diff
@@ -389,3 +389,81 @@ function reps_core_update_7012() {
     $source->startImport();
   }
 }
+
+/**
+ * Forcing reerting reps_core to apply upgrade 7014.
+ */
+function reps_core_update_7013() {
+  features_revert(
+    array(
+      'reps_core' => array(
+        'field_base',
+        'field_instance',
+      ),
+    )
+  );
+}
+
+/**
+ * Add value for existing entities delete unsused rightcol_position field.
+ */
+function reps_core_update_7014() {
+  // Set default value for field_reps_display_region in existing bean blocks.
+  $field_name = 'field_reps_display_region';
+  $entity_type = 'bean';
```

Comment:
Just an advise. No need to put every string in a variable as you overload php memory. The code look more understandable but there is not that much difference with `entity_load('bean');`. It's in a hook update here so it's not really important, but in a hook alter on every page for example, these small details have some impact. Basically, I make new variable if I'm using it at least twise, otherwise, no need.

Reply:
$entity_type is used at line 414 and 415